__tests__/app.test.js

```diff
@@ -29,6 +30,25 @@ describe("GET /api/topics", () => {
   });
 });
 
+describe("GET /api/", () => {
+    test("GET:200 - responds with a 200 status code and an object of availiable endpoint names", () => {
+      return request(app)
+        .get("/api/")
+        .expect(200)
+        .then(({_body}) => {
+          expect(_body.endpoints);
+          const endpointNames = Object.keys(_body.endpoints)
+          const endpointNamesRef = Object.keys(endpointsRef)
+          expect (endpointNames).toEqual(endpointNamesRef)
+          endpointNames.forEach((endpoint) => {
+            const endpointProperties = Object.keys(_body.endpoints[endpoint])
+            const endpointPropertiesRef = Object.keys(endpointsRef[endpoint])
+            expect (endpointProperties).toEqual(endpointPropertiesRef)
+          });
```

Comment:
Nicely done! You can simplify this further by just having the response equal the contents of the endpoints file if you wish. I'll leave it up to you if you wanna refactor. 🙂

Reply:
ah true, I think I wasn't to sure how deep that would check if I did it that way. hmm perhaps stringify can help. thanks